docs/book/src/topics/powervs/prerequisites.md

```diff
@@ -50,12 +50,13 @@ A public network is required for your kubernetes cluster. Perform the following
 
 ```shell
 $ export IBMCLOUD_API_KEY=<API_KEY>
-$ pvsadm image import --pvs-instance-id <SERVICE_INSTANCE_ID> -b <BUCKETNAME> --object <OBJECT> --pvs-image-name <POWERVS_IMAGE_NAME> --bucket-region <REGION> --public-bucket
+$ capibmadm powervs image import --service-instance-id <SERVICE_INSTANCE_ID>  --zone <ZONE> -r <REGION> --object <OBJECT>  --pvs-image-name <POWERVS_IMAGE_NAME> --public-bucket -b  <BUCKETNAME>
 ```
 
 e.g:
 ```shell
-$ pvsadm image import --pvs-instance-id 6d892c30-5387-4685-85d0-4999d9c22a8c -b power-oss-bucket --object capibm-powervs-centos-streams8-1-24-2.ova.gz --pvs-image-name capibm-powervs-centos-streams8-1-24-2 --bucket-region us-south --public-bucket
+$ capibmadm powervs image import --service-instance-id c6c6945a-b6c6-4283-b0a9-14a702bdc764  --zone syd04 -r us-south --object rhcos-412.86.202208090152-0-powervs.ppc64le.ova.gz  --pvs-image-name test_image_new11 --public-bucket -b  ocp-development-public-bucket
```

Comment:
```suggestion
$ capibmadm powervs image import --service-instance-id 3229a94c-af54-4212-bf60-6202b6fd0a07  --zone osa21 --bucket-region jp-tok --object RHEL9.0-image.ova.gz  --name powervs_image --public-bucket -b  ocp-development-public-bucket
```

Reply:
@irapandey Please update this as well to use --bucket-region instead of -r and --powervs-image-name with -name as suggested above